cypress.json

```diff
@@ -1,4 +1,5 @@
 {
   "baseUrl": "http://localhost:3001",
-  "fixturesFolder": false
+  "fixturesFolder": false,
+  "chromeWebSecurity": false
```

Comment:
this we can remove once `app.scope.dev` is prepared for cors preflight of cypress